Firestore/Source/Util/FSTLogger.m

```diff
@@ -16,7 +16,12 @@
 
 #import "Firestore/Source/Util/FSTLogger.h"
 
-#import "FIRLogger.h"
+#ifdef COCOAPODS
+#import <FirebaseCore/FIRLogger.h>
```

Comment:
It should only be the #import <FirebaseCore/FIRLogger.h> line in github. copybara will add the ifdef wrapper for the internal version.

Reply:
Fixed.